### DISCUSSION TITLE
Luz nos Olhos

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#315] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/luz-nos-olhos/

**Comment by brunoaugusto** [comment]
Adoro que mesmo mais de 10 anos depois Skyrim ainda continua vivo assim :smiley:

**Comment by one-eyed-willy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [terça-feira, 17 de maio de 2022](<https://www.vish.blog.br/imagens/tirinhas/luz-nos-olhos/#comment_1128936393>):

> Adoro que mesmo mais de 10 anos depois Skyrim ainda continua vivo assim 😃

Às vezes eu fico tentado a baixar, instalar todos os *mods* (porque Skyrim *vanilla* é feio bagarai) e jogar de novo. Mas daí eu lembro de como é demorado de finalizar "pra valer" e a preguiça vence
